brion/types.h

```diff
@@ -40,7 +40,7 @@
 #  include <vmmlib/vmmlib.hpp>
 #pragma warning(pop)
 
-/** @namespace brion Blue Brain File IO classes */
+/** @namespace brion Blue Brain File data model classes, types and functions. */
```

Comment:
BlueBrain or Blue Brain ? 99% of the time I am seeing BlueBrain :) not an issue though